<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	background-color: &quot;#06402B&quot;
}</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>701</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>634</width>
       <height>372</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="horizontalSpacing">
       <number>-1</number>
      </property>
      <item row="1" column="3">
       <widget class="CardLabel" name="card24">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string>c24</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="CardLabel" name="card12">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string>c12</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="CardLabel" name="card21">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string>c21</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="CardLabel" name="card23">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string>c23</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="CardLabel" name="card14">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string>c14</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="CardLabel" name="card11">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string>c11</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="CardLabel" name="card22">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string>c22</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="CardLabel" name="card13">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string>c13</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="CardLabel" name="card25">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string>c25</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="CardLabel" name="card15">
        <property name="text">
         <string>c15</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>440</y>
       <width>571</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,2">
      <item>
       <widget class="QPushButton" name="StartBtn">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="NextBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Next Round</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ScoreLbl">
        <property name="text">
         <string>You: 0 - Computer: 0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="WelcomeLbl">
        <property name="text">
         <string>Click &quot;Start&quot; to begin</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>43</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CardLabel</class>
   <extends>QLabel</extends>
   <header>cardlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
